changelog/scripts/changelog.py

```diff
@@ -0,0 +1,48 @@
+import sys
+from os import listdir
+from os.path import getsize, isfile, join
+
+
+def main(argv):
+    if "release/" in argv[0]:
+        file_list = sorted(
+            [f for f in listdir("./changelog") if isfile(join("./changelog", f))]
+        )
+        file_list.reverse()
```

Comment:
just an fyi that there is also `sorted(..., reverse=True)`, not that it makes a difference!